spec/reek/smells/smell_warning_spec.rb

```diff
@@ -110,7 +120,15 @@
 
     it 'does not match on different params' do
       expect(uncommunicative.matches?(:UncommunicativeVariableName,
-                                      test: 'something else')).to be_falsy
+                                      parameters: {
+                                        test: 'something else'
+                                      })).to be_falsy
+    end
+
+    it 'does not match on different vars' do
+      expect(uncommunicative.matches?(:UncommunicativeVariableName,
+                                      parameters: { test: 'something' },
+                                      message: 'nothing')).to be_falsy
     end
```

Comment:
SmellWarning#matches_attributes? from here https://github.com/troessner/reek/pull/732/files#diff-4c9efdbf77261fe0d68c190a1db4e3fcR58 is a very powerful method now that is not spec'ed at all it sees. Can you please add some specs?